src/commands/relay.mjs

```diff
@@ -70,8 +72,8 @@ export class RelayCommand extends BaseCommand {
     }
 
     nodeIDs.forEach(nodeID => {
-      const networkKey = `network-${nodeID.trim()}-0:50211`
-      valuesArg += ` --set config.HEDERA_NETWORK.${networkKey}=0.0.3`
+      const networkKey = `network-${nodeID.trim()}:50211`
+      valuesArg += ` --set config.HEDERA_NETWORK='{"${networkKey}":"0.0.3"}'`
```

Comment:
It needs to be different account for different node. 

This is where we currently associate a node to an account: https://github.com/hashgraph/solo/blob/main/src/core/profile_manager.mjs#L151

We need to do similar here. So probably we need to create a helper function that returns a map of nodeID -> accountID that can be used both places.

Reply:
Updated